stroeer/core/v1/article.proto

```diff
@@ -961,6 +962,54 @@ message Body {
   repeated BodyNode children = 1;
 }
 
+// todo: rename to `Body`
+message BodyPart {
+  /*
+  * ### `types`
+  * | type | description |
+  * | -----| ------------|
+  * | `body` | formatted text and inlined elements |
```

Comment:
`Body.type = 'body'` is a bit weird. This should be the regular body text

Reply:
we should still keep in mind that there may be paginated articles, so `type` may not be unique.

Reply:
argh good point

Reply:
is pagination a blocker here? 

from the top of my head: is it possible to include all pages of an article inside the `body` part including some page break identifiers which are then evaluated in the pages api (like remove unecessary parts for the requested page)?

Reply:
if i understand correctly, the mentioned thingies like toc, disclaimer etc are placed conventionally by the frontend. doesn't that make them eligible to be pulled outside of the body?
this could make paging also easier imo.